plugins/MauticFocusBundle/Translations/en_US/messages.ini

```
@@ -109,3 +109,7 @@ mautic.campaign.focus.show="Focus Item"
 mautic.focus.form.confirmdelete="Delete the Focus item, %name%?"
 mautic.focus.protocol.mismatch="This site cannot be previewed because it is not secure. Please try using %url%."
 mautic.focus.blocking.iframe.header="This website blocks iframe previews with the %header% header. Please try another URL."
+mautic.config.tab.focusconfig = "Focus Settings"
+mautic.page.config.form.focus.pixel.enabled = "Enable Tracking-Pixel for Focus Items"
+mautic.page.config.form.focus.pixel = "Focus Item"
+mautic.page.config.form.focus.pixel.enabled.tooltip = "If set to no, there will be no Tracking Cookies set. This allows Focus Items to be used even if a Consent Manager is used and the user has denied consent to tracking, marketing cookies etc. The mautic_focus cookie will still be set (can be defined as "technically required") but nothing else."
```

The inner double quotes will have to be escaped or changed to single quotes.

```suggestion
mautic.page.config.form.focus.pixel.enabled.tooltip = "If set to no, there will be no Tracking Cookies set. This allows Focus Items to be used even if a Consent Manager is used and the user has denied consent to tracking, marketing cookies etc. The mautic_focus cookie will still be set (can be defined as \"technically required\") but nothing else."
```